site/en/docs/webstore/cws-dashboard-distribution/index.md

```diff
@@ -1,17 +1,19 @@
 ---
 layout: "layouts/doc-post.njk"
-title: "Prepare to publish: set up pricing and distribution"
-#date: TODO
-#updated: TODO
+title: "Prepare to publish: set up payment and distribution"
+date: 2021-07-29
```

Comment:
I'm sure this file must be older than that. It looks like the history got messed up by a rebase or something. Not sure what to do about this.

Reply:
Added the same date based on PR #450